pom.xml

```diff
@@ -105,6 +105,19 @@
 						<additionalparam>-Xdoclint:none</additionalparam>
 					</configuration>
 				</plugin>
+				<plugin>
+					<groupId>org.codehaus.mojo</groupId>
+					<artifactId>cobertura-maven-plugin</artifactId>
+					<version>2.7</version>
+					<inherited>true</inherited>
+					<configuration>
+						<formats>
+							<format>html</format>
+							<format>xml</format>
+						</formats>
+						<aggregate>true</aggregate>
+					</configuration>
+				</plugin>
```

Comment:
Not really required - We could remove Corbetura.

Reply:
Okay. Will update MR.